alf/environments/carla_sensors.py

```diff
@@ -567,10 +567,14 @@ def render(self, display):
             height, width = self._image.shape[1:3]
             image = np.transpose(self._image, (2, 1, 0))
             if width < MINIMUM_RENDER_WIDTH:
-                height = height * MINIMUM_RENDER_WIDTH // width
+                scaling_factor = max(
+                    float(MINIMUM_RENDER_HEIGHT) / height,
```

Comment:
if height is very small, scaling_factor will be very big. and width will be very big.

Reply:
I was thinking to keep the aspect ratio unchanged. But it might not be good for visualization in some cases as you suggested.
Now the rendering surface part has been rolled back. And the camera image scaling part is updated to be consistent.

Reply:
Updated. Now still keep the aspect ratio and added upper bound for scaled sizes.